spec/helpers/authoring.ts

```diff
@@ -194,7 +194,7 @@ class Authoring {
         this.save_button = element(by.buttonText('Save'));
         this.edit_button = element(by.id('Edit'));
         this.update_button = element(by.buttonText('Update'));
-        this.edit_correct_button = element(by.buttonText('Correct'));
+        this.edit_correct_button = element(by.css('[title="Correct"]'));
         this.edit_kill_button = element(by.buttonText('Kill'));
```

Comment:
seems like kill has the same issue

Reply:
It was not failing first time. I added change for all buttons that are in header